File: RepoTransBench/target_projects/Python/C++/scrapy_cssselect/public_tests/test_public_api_imports.cpp
#include <gtest/gtest.h>
#include "../src/dummy_cssselect.h"

TEST(TestApiImportsPublic, ImportSomeAttrsPublic) {
    for (const auto& attr : {
             "GenericTranslator",
             "parse",
             "SelectorError",
             "Selector",
             "HTMLTranslator",
         }) {
        EXPECT_TRUE(cssselect::hasattr(attr));
    }
}

TEST(TestApiImportsPublic, VersionPublic) {
    EXPECT_NE(cssselect::VERSION, "");
    EXPECT_TRUE(!cssselect::__version__.empty());
    EXPECT_GT(cssselect::__version__.length(), 0u);
}

File: RepoTransBench/target_projects/Python/Rust/scrapy_cssselect/public_tests/test_public_api_imports.rs
#[test]
fn test_import_some_attrs_public() {
    // Choose different subset of attributes and their presence
    use scrapy_cssselect_rs as cssselect;
    let _ = cssselect::GenericTranslator;
    let _ = cssselect::parse;
    let _ = cssselect::SelectorError;
    let _ = cssselect::Selector;
    let _ = cssselect::HTMLTranslator;
    // Ensured type exists
}

#[test]
fn test_version_public() {
    use scrapy_cssselect_rs as cssselect;
    assert!(!cssselect::VERSION.is_empty());
    assert!(cssselect::__version__.is_ascii());
    assert!(cssselect::__version__.len() > 0);
}